src/main/java/aztech/modern_industrialization/machines/blockentities/hatches/EnergyHatch.java

```diff
@@ -55,7 +57,8 @@ public EnergyHatch(BEP bep, String name, boolean input, CableTier tier) {
         this.registerComponents(energy);
     }
 
-    private final boolean input;
+    protected final boolean input;
```

Comment:
Any reason not to keep them `private`?

Reply:
Whoops, missed that one

Reply:
Whats the reason for these to be private when the other fields are protected, out of curiosity?

Reply:
I think you're asking the wrong question, and that the right question would be: Why are the others `protected` even though they don't seem to be accessed from outside the class? 😄 No idea though

Reply:
Fair enough. Trying to think of a situation where I may want to extend EnergyHatch and use those fields, I don't think there's really any that don't have alternatives. I can swap these to private and come back in the future if I wind up needing it for some reason.